examples/complete/main.tf

```diff
@@ -236,4 +240,5 @@ module "secret_manager_service_credential" {
   service_credentials_source_service_crn      = module.cloud_object_storage.cos_instance_id
   service_credentials_source_service_role_crn = "crn:v1:bluemix:public:iam::::serviceRole:Writer"
   service_credentials_parameters              = { "service-endpoints" : "public" }
+  custom_metadata                             = { "service-endpoints" : "public" } # can add any custom metadata here
```

Comment:
This value doesn't make sense when we already use it in `service_credentials_parameters` above. Just add any other custom metadata here

Reply:
@alex-reiff this one too